tracer/test/test-applications/integrations/Samples.Remoting/Samples.Remoting.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- override to remove core -->
+    <TargetFrameworks>net462</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System.Runtime.Remoting" />
```

Comment:
I'm kind of surprised our public referenced assemblies hadn't changed with this integration, then I realised Datadog.Trace.Vendors.Serilog.Context.LogContext references System.Runtime.Remoting 😱

Reply:
Oh right, before `AsyncLocal` you would want to use `System.Runtime.Remoting.Messaging.LogicalCallContext` to store information in the current ExecutionContext. Wait, we should be able to get rid of that now right??